.github/workflows/azureml-unit-tests.yml

```diff
@@ -6,13 +6,12 @@
 name: azureml-unit-tests
 
 on:
-  pull_request:
-    branches: [ staging, main ]
-
-  # push:
-  #   # because we can't schedule runs for non-main branches,
-  #   # to ensure we are running the build on the staging branch, we can add push policy for it
-  #   branches: [pradjoshi/update_azureml_cluster]
+  pull_request_target:
+    types:
+      - opened
+    branches:
+      - 'staging'
+      - 'main'
```

Comment:
Maybe it would be good to add a comment here explaining how using this `pull_request_target` helps us with the forked PR. What do you think?

Reply:
Comment added.